core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -709,6 +709,7 @@ private[spark] class BlockManager(
       level: StorageLevel,
       tellMaster: Boolean = true): Boolean = {
     require(bytes != null, "Bytes is null")
+    logInfo(s"Putting block $blockId with the size of ${bytes.size} bytes in ${level.description}")
```

Comment:
I think this logs every single put of bytes, not just shuffled blocks. It would have to be debug level, but not sure this is the right place